comeet/UI/MainMenu/MainMenuViewController.swift

```diff
@@ -89,6 +89,15 @@ class MainMenuViewController: BaseViewController, WWCalendarTimeSelectorProtocol
                 print("")
                 return
         }
+        
+
+        if identifier == "toAgenda" {
+            let dest = segue.destination as? MyAgendaViewController
+            dest?.viewModel = viewModel
```

Comment:
We are hooking up the viewModels and other properties of the flow in the Router.